lib/Sandbox.js

```diff
@@ -59,6 +59,14 @@ export type EnvironmentVarExport = {
   __BUILT_IN_DO_NOT_USE_OR_YOU_WILL_BE_PIPd?: boolean;
 };
 
+export type EsyConfig = {
+  build: ?string;
+  buildOutOfSource: boolean;
```

Comment:
I tend to prefer these "presence" booleans to be such that absent implies `false`. So in that case, it would probably be named something like `buildsInSource`.